### PR TITLE
CI: Add Release Workflow

### PR DESCRIPTION
**Pull request by MasonT8198**
This is my first time ever writing a workflow tbh, so I may need some help for this one lol

### CODE REVIEW

**Comment by MasonT8198** [comment]
I've spent like 80+ tries over yesterday and today to get this thing to upload correctly, seems I may be too inept to finish this, it only needs like 2 tweaks and it'll work properly

**Comment by MasonT8198** [comment]
As much as I hate to give up on something, I've tried plenty of different methods to upload this release, build tags work and the release code is right, I just can't find the path of the build artifacts I was working with, however, if the right individual wanted to finish this, it wouldn't be too hard to do. I will still be working on PR #49 however.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -84,3 +102,48 @@ jobs:
       with:
         name: extract-xiso_${{ runner.os }}
         path: artifacts
+        
+  Release:
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/xemu-v'))
```
Comment (JayFoxRox) [inline .github/workflows/CI.yml:107]: If anyone continues on this: This mentions xemu, when it definitely shouldn't.

Comment (MasonT8198) [reply .github/workflows/CI.yml:107]: Yes, I tried to use some of the build workflow from the xemu project as a start as I'm new to using a workflow, I removed this in my local build but must've forgot to remove it here, This should be the only reference to xemu however

**Pull request closed without merging**